bin/build/src/release/elastic_beanstalk.clj

```diff
@@ -94,9 +94,9 @@
   (u/step (format "Create launch-aws-eb.html for Docker image %s" (c/docker-tag))
     (u/delete-file-if-exists! html-file-path)
     (spit html-file-path
-          (stencil/render-file launch-template-filename
-                               {:url (java.net.URLEncoder/encode (c/artifact-download-url "metabase-aws-eb.zip")
-                                                                 "UTF-8")}))
+          (stencil/render (slurp launch-template-filename)
```

Comment:
stencil is expecting a path relative to the classpath, when I reworked this last night I changed it to an absolute path, and it didn't like that. This was the fix